cmd/storage.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/storage/cp"
+	"github.com/supabase/cli/internal/storage/ls"
+	"github.com/supabase/cli/internal/storage/mv"
+	"github.com/supabase/cli/internal/storage/rm"
+)
+
+var (
+	storageCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "storage",
+		Short:   "Manage Supabase Storage objects",
+	}
+
+	recursive bool
+
+	lsCmd = &cobra.Command{
+		Use:     "ls [prefix]",
```

Comment:
```suggestion
		Use:     "ls [path]",
```

As this argument overrides whole `objectPath`, not appends it

Reply:
hmm I named this `prefix` because it can match a partial path prefix. But I guess `path` is fine.